pkgs/tools/nix/options/options

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+TEMPFILE=`mktemp`.nix
+
+echo '
+let
+    flake-compat = builtins.fetchTarball {
+        url = "https://github.com/edolstra/flake-compat/archive/master.tar.gz";
```

Comment:
Wouldn't it be better to pin this to a commit and add `sha256` attribute to make sure this can be cached?

Reply:
Also, this would ensure that this command wouldn't break if `flake-compat` broke something, for example.

Reply:
And it wouldn't block the evaluator. A fixed-output derivation produces a Nixlang value orders of magnitude faster than actual fetching.